packages/mui-base/src/TablePaginationUnstyled/TablePaginationUnstyled.types.ts

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 import { OverrideProps } from '@mui/types';
 import { SlotComponentProps } from '../utils';
+import TablePaginationActionsUnstyled from './TablePaginationActionsUnstyled';
+import * as CommonTypes from './common.types';
```

Comment:
Doesn't `import { ItemAriaLabelType } from './common.types'` work? It would be more inline with our other imports.

Reply:
It doesn't have to be, does it?
If you `import { ItemAriaLabelType } from './common.types';` line 14 can be deleted. Or am I missing something?